src/run/index.js

```diff
@@ -29,6 +30,8 @@ export default async function (opts) {
   const events = load.events(name, opts.event)
   const [ fileName, handler ] = lambdaConfig.Handler.split('.')
 
+  merge(process.env, load.envVars(env))
```

Comment:
This will overwrite env vars on the command line? `DB_TABLE=custom shep run foo` would still use whatever is specified in the .json file and not the CLI supplied one